docs/source/algorithms/FilterEvents-v1.rst

```diff
@@ -100,6 +100,17 @@ name ended with '\_unfiltered'.
 If input property 'OutputWorkspaceIndexedFrom1' is set to True, then
 this workspace shall not be outputed.
 
+Using FilterEvents with fast-changing logs
+##########################################
+
+There are a few parameters to consider when the log filtering is expected to produce a large
+splitter table. An example of such a case would be a data file for which the events need to be split
+according to a log with two or more states changing in the kHz range. To reduce the filtering time,
+one may do the following:
+
+ - Make sure the ``SplitterWorkspace`` input is a :ref:`MatrixWorkspace <MatrixWorkspace>`. Such a workspace can be produced by using the ``FastLog = True`` option when calling :ref:`GenerateEventsFilter <algm-GenerateEventsFilter>`.
```

Comment:
You need to remove the whitespace in front of your bulleted list items (no indentation).